finetuner-workflow/finetuner/requirements.txt

```diff
@@ -8,3 +8,4 @@ accelerate~=0.17.1
 tensorizer==1.0.1
 einops
 flash-attn
+kserve
```

Comment:
Few things:

1. We should include some sort of versioning for all the packages. That way everytime a customer builds an image going through the docs, we know it'll be consistent (as much as we can control at least)

2. The pip install fails both locally and in the docker build. Were you able to build the image and test it in the workflow? In the workflow it's still using the goose finetuner image.

3. Should we be bumping tensorizer to the newest version, 1.1.0?